TurboWarp/Test/Test.js

```diff
@@ -0,0 +1,619 @@
door [REPLACE]","_replace regex /[REGEX]/[FLAGS] in [STRING] with [REPLACE]":"vervang regex /[REGEX]/[FLAGS] in [STRING] door [REPLACE]","_unicode [NUM] as letter":"unicode [NUM] als letter","_unicode of [STRING]":"unicode van [STRING]"},"pt-br":{"_Text":"Texto"},"ru":{"_Exactly Title Case":"Настоящий Тайтл Кейс","_MiXeD CaSe":"СмЕшАнНыЙ КеЙс","_Text":"Текст","_Title Case":"Тайтл Кейс","_UPPERCASE":"ВЕРХНИЙ РЕГИСТР","_[STRING] matches regex /[REGEX]/[FLAGS]?":"[STRING] совпадает с regex'ом /[REGEX]/[FLAGS]?","_convert [STRING] to [TEXTCASE]":"сконвертировать строку [STRING] в [TEXTCASE]","_count [SUBSTRING] in [STRING]":"количество [SUBSTRING] в [STRING]","_count regex /[REGEX]/[FLAGS] in [STRING]":"посчитать regex /[REGEX]/[FLAGS] в [STRING]","_index of [SUBSTRING] in [STRING]":"индекс [SUBSTRING] в [STRING]","_is [OPERAND1] identical to [OPERAND2]?":"[OPERAND1] идентичный с [OPERAND2]?","_is [STRING] [TEXTCASE]?":"строка [STRING] это [TEXTCASE]?","_item [ITEM] of [STRING] matched by regex /[REGEX]/[FLAGS]":"предмет [ITEM] строки [STRING] совпадающий regex'ом /[REGEX]/[FLAGS]","_item [ITEM] of [STRING] split by [SPLIT]":"предмет [ITEM] строки [STRING] распределённый [SPLIT]","_letters [LETTER1] to [LETTER2] of [STRING]":"буквы с [LETTER1] до [LETTER2] строки [STRING]","_lowercase":"нижний регистр","_repeat [STRING] [REPEAT] times":"повторить [STRING] [REPEAT] раз","_replace [SUBSTRING] in [STRING] with [REPLACE]":"заменить [SUBSTRING] в [STRING] на [REPLACE]","_replace regex /[REGEX]/[FLAGS] in [STRING] with [REPLACE]":"заменить regex /[REGEX]/[FLAGS] в [STRING] на [REPLACE]","_unicode [NUM] as letter":"юникод [NUM] как букву","_unicode of [STRING]":"юникод [STRING]"},"tr":{"_Text":"Metin"},"uk":{"_Text":"Текст"},"zh-cn":{"_Exactly Title Case":"精确标题","_MiXeD CaSe":"混合大小写","_Text":"文本","_Title Case":"标题","_UPPERCASE":"大写","_[STRING] matches regex /[REGEX]/[FLAGS]?":"[STRING] 满足正则表达式 /[REGEX]/[FLAGS]？","_convert [STRING] to [TEXTCASE]":"转换[STRING]为[TEXTCASE]","_count [SUBSTRING] in [STRING]":"[STRING]中[SUBSTRING]的数量","_count regex /[REGEX]/[FLAGS] in [STRING]":"使用正则表达式 /[REGEX]/[FLAGS] 在 [STRING] 匹配的数量","_index of [SUBSTRING] in [STRING]":"[STRING]中[SUBSTRING]的位置","_is [OPERAND1] identical to [OPERAND2]?":"[OPERAND1]===[OPERAND2]","_is [STRING] [TEXTCASE]?":"[STRING]是[TEXTCASE]？","_item [ITEM] of [STRING] matched by regex /[REGEX]/[FLAGS]":"使用正则表达式 /[REGEX]/[FLAGS]匹配[STRING]的第[ITEM]个内容","_item [ITEM] of [STRING] split by [SPLIT]":"以[SPLIT]分割[STRING]后的第[ITEM]项","_letters [LETTER1] to [LETTER2] of [STRING]":"[STRING]的第[LETTER1]到第[LETTER2]位","_lowercase":"小写","_repeat [STRING] [REPEAT] times":"重复[REPEAT]个[STRING]","_replace [SUBSTRING] in [STRING] with [REPLACE]":"替换[STRING]中的[SUBSTRING]为[REPLACE]","_replace regex /[REGEX]/[FLAGS] in [STRING] with [REPLACE]":"使用正则表达式 /[REGEX]/[FLAGS] 在 [STRING]中替换为  [REPLACE]","_unicode [NUM] as letter":"unicode[NUM]对应的字符","_unicode of [STRING]":"[STRING]的unicode"}});/* end generated l10n code */(function (Scratch) {
```

Comment:
Currently Gandi-IDE only supports zh-cn and en, it is recommended to remove redundant I10n configurations in order to reduce the file size.